rpm/msr-safe.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -o nounset
+set -o pipefail
+
+. /etc/sysconfig/msr-safe
+
+start() {
+  if [ -z "${WL_CPU:-}" ]; then
+    WL_CPU=$(printf 'wl_%.2x%x\n' $(lscpu | grep "CPU family:" | awk -F: '{print $2}') $(lscpu | grep "Model:" | awk -F: '{print $2}'))
```

Comment:
Suggestion to avoid lscpu dependency to get the white list name:

function wl_name () {
    printf 'wl_%.2x%x\n' \
        $(sed 's|[\t ]*:|:|' /proc/cpuinfo | grep 'cpu family:' | head -n1 | awk -F: '{print $2}') \
        $(sed 's|[\t ]*:|:|' /proc/cpuinfo | grep 'model:' | head -n1 | awk -F: '{print $2}')
}